### PR TITLE
[MRG] FastText out-of-vocab fix 

### PR DESCRIPTION
**Pull request by jayantj**
1. Cleans up some old tests for FastText wrapper
2. Adds (initially failing) tests for ensuring vectors for out-of-vocab words are as expected.
3. Fixes in FastText wrapper with correct out-of-vocab word handling.

@menshikh-iv @prakhar2b Please review.

### CODE REVIEW

**Comment by prakhar2b** [comment]
@jayantj That's great. Thanks :smile: 
I'll update PR #1341 accordingly now.

File: gensim/models/wrappers/fasttext.py [hunk 1]
```diff
@@ -344,7 +344,7 @@ def init_ngrams(self):
         ngram_indices = []
         for i, ngram in enumerate(all_ngrams):
             ngram_hash = self.ft_hash(ngram)
-            ngram_indices.append((len(self.wv.vocab) + ngram_hash) % self.bucket)
+            ngram_indices.append((len(self.wv.vocab)) + ngram_hash % self.bucket)
```
Comment (menshikh-iv) [inline gensim/models/wrappers/fasttext.py]: No needed brackets `(len(...))`
Also, you change operation order, it's correct behavior?

Comment (jayantj) [inline gensim/models/wrappers/fasttext.py]: Removed the extra brackets.
Yes, the new operation order is current behaviour, it was a bug earlier.

**Comment by menshikh-iv** [comment]
Nice work @jayantj :+1:

**Pull request merged**